src/lib/KStorage.ts

```diff
@@ -85,8 +92,22 @@ export class KStorage {
 	}
 
 	close() {
+    this._subscription.unsubscribe();
 	  return Promise.resolve(true);
 	}
+
+  next({ key, value }) {
```

Comment:
Indentation looks out here, and missing return type and doc block comments

Reply:
Added doc / return type. Using vim, I'm not seeing the same indentation issue as shown on GitHub. Could you check it locally as well?